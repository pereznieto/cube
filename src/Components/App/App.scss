@import '../../styles/main';

.cube-app {
  .master-wrapper {
    position: relative;
    max-width: width(xl);
    margin: 20px auto;
    color: $primary-black;
    font-family: $sans-font;
    text-align: center;
  }
}
