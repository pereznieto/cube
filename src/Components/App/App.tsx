import * as React from 'react';
import TimerWrapper from '../TimerWrapper';
import styles from './App.scss';

const App = () => (
  <div className={styles['cube-app']} >
    <div className={styles['master-wrapper']}>
      <h1>Rubik's Cube Timer</h1>
      <TimerWrapper />
    </div>
  </div>
);

export default App;
