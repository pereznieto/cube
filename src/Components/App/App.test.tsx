import { shallow } from 'enzyme';
import * as React from 'react';
import App from './App';

describe('App', () => {
  it('should render without crashing', () => {
    shallow(<App />);
  });

  it('should render App as expected', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
