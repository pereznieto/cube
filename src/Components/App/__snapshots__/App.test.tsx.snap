// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render App as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className={undefined}
      >
        <h1>
          Rubik's Cube Timer
        </h1>
        <TimerWrapper />
      </div>,
      "className": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Rubik's Cube Timer
          </h1>,
          <TimerWrapper />,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Rubik's Cube Timer",
          },
          "ref": null,
          "rendered": "Rubik's Cube Timer",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className={undefined}
        >
          <h1>
            Rubik's Cube Timer
          </h1>
          <TimerWrapper />
        </div>,
        "className": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Rubik's Cube Timer
            </h1>,
            <TimerWrapper />,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Rubik's Cube Timer",
            },
            "ref": null,
            "rendered": "Rubik's Cube Timer",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
